avm/src/lib.rs

```diff
@@ -49,11 +49,13 @@ pub fn use_version(version: &Version) -> Result<()> {
     let installed_versions = read_installed_versions();
     // Make sure the requested version is installed
     if !installed_versions.contains(version) {
-        println!(
-            "Version {} is not installed, staying on version {}.",
-            version,
-            current_version()?
-        );
+        let current = current_version();
+        if !current.is_err() {
```

Comment:
you can simplify this with 
```rust
if let Ok(current) = current_version()? {
...
}
```

but I think we can still give some info even if there's no current version yet so a `match` on `current_version` sounds good where you still print `Version {} is not installed` if there's no current version

Reply:
yeah - more info helps those that are at my level and need to read it :+1: